ethers-solc/src/compile/mod.rs

```diff
@@ -686,6 +686,7 @@ fn version_from_output(output: Output) -> Result<Version> {
         let version = output
             .stdout
             .lines()
+            .filter(|l| !l.as_ref().unwrap().is_empty())
```

Comment:
great,

now that I'm seeing this again, we can 
```suggestion
            .filter_map(Result::ok)
            .filter(|l| !l.trim().is_empty())
```
and get rid of the `map_err` entirely